src/TerminalSettings/TerminalSettings/MainPage.cpp

```diff
@@ -56,39 +56,39 @@ namespace winrt::TerminalSettings::implementation
 
     }
 
-    void MainPage::SettingsNav_ItemInvoked(Controls::NavigationView sender, Controls::NavigationViewItemInvokedEventArgs args)
+    void winrt::TerminalSettings::implementation::MainPage::SettingsNav_SelectionChanged_1(winrt::Microsoft::UI::Xaml::Controls::NavigationView const& sender, winrt::Microsoft::UI::Xaml::Controls::NavigationViewSelectionChangedEventArgs const& args)

```

Comment:
```suggestion
    void MainPage::SettingsNav_SelectionChanged_1(NavigationView const& sender, NavigationViewSelectionChangedEventArgs const& args)
```
And on line 10, replace
```c++
using namespace Windows::UI::Xaml;
```
with
```c++
using namespace winrt::Microsoft::UI::Xaml::Controls;
using namespace winrt::Windows::UI::Xaml;
```